src/app/ReadClient.h

```diff
@@ -345,6 +357,18 @@ class ReadClient : public Messaging::ExchangeDelegate
     InteractionModelEngine * mpImEngine = nullptr;
     ReadPrepareParams mReadPrepareParams;
     uint32_t mNumRetries = 0;
+
+    // Reserved size for the FabricFiltered boolean flag, which takes up 1 byte for the control tag and 1 byte for the context tag.
+    static constexpr uint16_t kFabricFiltered = 1 + 1;
```

Comment:
```suggestion
    static constexpr uint16_t kReservedSizeForFabricFiltered = 1 + 1;
```

Reply:
removed because I have moved the data version filter encoding to the end for read/subscribe request following the spec.